web/modules/custom/instaform/src/Form/UserEditForm.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+namespace Drupal\instaform\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Messenger\MessengerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Url;
+use Drupal\Component\Serialization\Json;
+use Drupal\Core\TempStore\PrivateTempStoreFactory;
+
+/**
+ * A class to create form to edit user's data.
+ */
+class UserEditForm extends FormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(MessengerInterface $msg, AccountInterface $account, EntityTypeManagerInterface $entity_type_manager, PrivateTempStoreFactory $tempstore_private) {
+    $this->messenger = $msg;
+    $this->account = $account;
+    $this->entityTypeManager = $entity_type_manager;
+    $this->tempstore_private = $tempstore_private;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('messenger'),
+      $container->get('current_user'),
+      $container->get('entity_type.manager'),
+      $container->get('tempstore.private')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'usereditform';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $uid = $this->account->id();
+    if ($uid) {
+      $user = $this->entityTypeManager->getStorage('user')->load($uid);
+      $name = $user->get('name')->value;
+      $fullname = $user->get('field_full_name')->value;
+      $bio = $user->get('field_bio')->value;
+      $mail = $user->get('mail')->value;
+
+      $phone = $user->get('field_phone')->value;
+      $website = $user->get('field_website')->value;
+
+      $pictureUri = $user->user_picture->entity->getFileUri();
```

Comment:
DIrectly we can fetch user profile image so that we don't need to use `str_replace` afterwards